docs/topics/proc_git-workflow-for-making-changes.adoc

```diff
@@ -34,9 +34,14 @@ If you are unsure, ask someone from the team for help.
 --
 .. Label the pull request appropriately. For more information about labels, see xref:using-github-labels_{context}[].
 .. Reference the issue that your pull request resolves in the description, using the formulation `resolves $ID`, where `$ID` is the numerical ID of the issue. Doing this automatically closes the issue when the pull request is accepted.
-
+--
+. After your pull request is created, include a link to the edited document in a comment.
++
+--
+.. Preview the rendered documentation with the changes from the pull request by navigating to the `Conversation` tab of the pull request, clicking _Show All Checks_, and _Details_ next to the revealed `deploy/netlify`.
```

Comment:
`Conversation` should be _Conversation_, also, you could probably omit _the revealed_ form the sentence, and it would IMHO work, too:)
 
 Otherwise, changes look good to me:)